src/main/User/CreateDeveloper.java

```diff
@@ -0,0 +1,67 @@
+package User;
+
+import Config.MongoConfig;
+import Security.SecurityUtils;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+import static com.mongodb.client.model.Filters.eq;
+
+public class CreateDeveloper {
+
+  public static void main(String[] args) throws Validation.ValidationException {
+    MongoClient client = MongoConfig.getMongoClient();
+    MongoDatabase db = client.getDatabase(MongoConfig.getDatabaseName());
+    SecurityUtils securityUtils = new SecurityUtils();
+
+    String firstName = "Steffen";
+    String lastName = "Cornwell";
+    String birthDate = "07-25-1998";
+    String email = "steffencornwell@gmail.com";
+    String phone = "4103020506";
+    String address = "7145 Deer Valley Road";
+    String city = "Highland";
+    String state = "MD";
+    String zipcode = "20777";
+    Boolean twoFactorOn = false;
+    String username = "STEFFEN-CORNWELL-DEVELOPER-2";
```

Comment:
why is this all hardcoded

Reply:
Done, made it into a scanner prompt when you run it. It is intended not to be run on the server, but on a local computer